e2e_tests/tests/cluster/test_users.py

```diff
@@ -946,30 +989,42 @@ def test_change_displayname(clean_auth: None) -> None:
     assert current_user is not None and current_user.id
 
     # Rename user using display name
-    patch_user = bindings.v1PatchUser(displayName="renamed display-name")
-    bindings.patch_PatchUser(sess, body=patch_user, userId=current_user.id)
+    patch_user = bindings.v1PatchUser(displayName="renamed")
```

Comment:
can we bring back the `renamed display-name` like before, so we have spaces and dashes?